### PR TITLE
Add label app=enmasse to all entities

### PR DESCRIPTION
**Pull request by dmvolod**
The activities with labels help to delete with selector like
oc delete all,pvc,configmap,rolebindings,secrets,sa -l 'app=enmasse'
I will update ansible configs and add remove action inside deploy.sh in the future if necessary

### CODE REVIEW

**Comment by dmvolod** [comment]
@lulf , added to .jsonnet files when was missing

File: templates/include/api-server.jsonnet [hunk 1]
```diff
@@ -145,7 +145,8 @@ local images = import "images.jsonnet";
         "apiVersion": "apiregistration.k8s.io/v1beta1",
         "kind": "APIService",
         "metadata": {
-          "name": "v1alpha1.enmasse.io"
+          "name": "v1alpha1.enmasse.io",
+          "app": "enmasse"
```
Comment (lulf) [inline templates/include/api-server.jsonnet]: Needs to be nested within a label

Comment (dmvolod) [reply templates/include/api-server.jsonnet]: sorry, fixed

**Comment by lulf** [comment]
@dmvolod Thanks!

**Pull request merged**